frontend_tests/node_tests/dispatch.js

```diff
@@ -872,8 +872,8 @@ with_overrides(function (override) {
         assert.equal(page_params[parameter_name], true);
     }
 
-    var event = event_fixtures.realm__update__create_stream_by_admins_only;
-    test_realm_boolean(event, 'realm_create_stream_by_admins_only');
```

Comment:
I wasn't sure about this being `test_realm_boolean` but since it seemed to work for `invite_to_stream_policy`, I've kept it that way.